tox.ini

```
@@ -937,52 +943,79 @@ commands =
-        CLEAR="\033[0m"; \
-        printf "\n${CYAN}---------------------------------------------------------\n" ;\
-        printf "Make sure to add the following to your /etc/hosts file:${CLEAR}\n";\
-        printf "127.0.0.1 k3d-registry.localhost\n";\
-        printf "${CYAN}---------------------------------------------------------${CLEAR}\n"'
+    bash -c 'k3d registry create registry.localhost --port 12345 -v $HOME/.k3d-registry:/var/lib/registry || true'
+    sudo {envpython} scripts/patch_hosts.py --add-k3d-registry --fix-docker-hosts
 
 [testenv:dev.k8s.start]
-changedir = {toxinidir}/packages/grid
+description = Start local Kubernetes registry & cluster with k3d
+changedir = {toxinidir}
+passenv=*
 allowlist_externals =
     bash
-    k3d
+    sleep
+    tox
 commands =
-    bash -c 'URL=http://k3d-registry.localhost:12345/v2/_catalog; \
-        curl -X GET $URL || (echo "ERROR: Add \"127.0.0.1 k3d-registry.localhost\" to /etc/hosts" && exit 1)'
+    tox -e dev.k8s.registry
+    sleep 3
+    bash -c 'URL=http://k3d-registry.localhost:12345/v2/_catalog; curl -X GET $URL'
     bash -c 'k3d cluster create syft-dev -p "8080:80@loadbalancer" --registry-use k3d-registry.localhost:12345; \
-        k3d cluster list'
+        kubectl create namespace syft;'
+    tox -e dev.k8s.info
 
 [testenv:dev.k8s.deploy]
+description = Deploy to a local Kubernetes cluster with Devspace
 changedir = {toxinidir}/packages/grid
 passenv=HOME, USER
 allowlist_externals =
     bash
 commands =
-    bash -c '\
-        devspace use context k3d-syft-dev; \
-        devspace use namespace syft; \
-        devspace deploy --var CONTAINER_REGISTRY=k3d-registry.localhost:12345 --config=devspace.yaml'
+    bash -c 'devspace deploy -b --kube-context k3d-syft-dev --namespace syft --var CONTAINER_REGISTRY=k3d-registry.localhost:12345'
+
+[testenv:dev.k8s.info]
+description = Gather info about the localKubernetes cluster
+passenv=HOME, USER
+ignore_errors = True
```

Q: Why do we ignore errors at this place?